test/e2e-helm/e2e_helm_metrics_test.go

```diff
@@ -0,0 +1,182 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e_helm_test
+
+import (
+	"encoding/base64"
+	"fmt"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo" //nolint:golint
+	. "github.com/onsi/gomega" //nolint:golint
+	kbtestutils "sigs.k8s.io/kubebuilder/test/e2e/utils"
+)
+
+var _ = Describe("Exporting metrics from a Helm Project", func() {
+
+	const curlImage = "curlimages/curl:latest"
+	var controllerPodName string
+
+	Context("built with operator-sdk", func() {
+		BeforeEach(func() {
+			By("Ensuring that the namespace is not created")
+			verifyControllerUp := func() error {
+				_, err := tc.Kubectl.Command("get", "namespace", tc.Kubectl.Namespace)
+				return err
+			}
+			Eventually(verifyControllerUp, time.Minute, time.Second).ShouldNot(Succeed())
```

Comment:
This section will succeed for any error. We should check that the error is specifically "namespace not found"